RELEASE_NOTES.rst

```diff
@@ -1,4 +1,139 @@
-coala 0.7.0 - baloo
+coala 0.8.0 - grizzly
+=====================
+
+::
+
+          _|\____/|_
+         //  \    /  \
+        //   o    o  |\________________
+       /|            |    \            \
+      | |      []    |     |       \    \
+      |  \   \____/ /       \       |    \
+      |   \________/         "      "     |
+      |                                   |
+      |                  \          \     |
+       \       \         |\          |    |
+        \      /\        | \         |    |
+        |     |  \       |  \        |     \
+        |     |   |     /    |      /\      |
+        |    /    |    |     |     /  |      \
+       /_____}   /_____}    /______}  {_______}
+
+^ Fabian, gimme something better than this :P
```

Comment:
he claims to have no time right now, but proposes to try to only make a claw. Maybe try to make a claw with the asciiart style of the previous release arts?